source/templates/writing-helpers.md

```diff
@@ -218,24 +218,44 @@ And two named arguments:
 * `format="YYY MM DD h:mm"`
 * `locale="en"`
 
-### Stateful Helpers
+### Service Helpers
```

Comment:
I was going with the terminology introduced in the original ticket (#464), but if is okay with @mmun I'd choose "Class-based helpers", since that is exactly what they are: Classy Helpers.